engine/crates/integration-tests/data/sse-subgraph/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM oven/bun:1.1.27-alpine
+
+RUN adduser -g wheel -D grafbase -h "/grafbase" && mkdir -p /grafbase && chown grafbase: /grafbase
```

Comment:
I'm not familiar with the colon in `chown grafbase: /grafbase` part, what does it do? (it's a case of RTFM but I'm lazy)

Reply:
I wasn't familiar either, this was a copy & paste from the gateway dockefile.

But looked it up: it sets the group to whatever the users login group is.

Reply:
TIL. Thanks!